cv32e40x/tests/cfg/default.yaml

```diff
@@ -11,6 +11,7 @@ plusargs: >
     +enable_zbb_extension=1
     +enable_zbc_extension=1
     +enable_zbs_extension=1
+cv_sw_march: rv32imc_zba1p00_zbb1p00_zbc1p00_zbs1p00
```

Comment:
With changes that were simultaneously done in another PR, I think this "cv_sw_march:" line got duplicated.

Reply:
Good catch, thanks!